README.md

```diff
@@ -421,11 +249,15 @@ Now you should be able to run Julia like this:
 
 On Windows, double-click `usr/bin/julia.exe`.
 
-If everything works correctly, you will see a Julia banner and an interactive prompt into which you can enter expressions for evaluation.
-You can read about [getting started](https://julialang.org/manual/getting-started) in the manual.
+If everything works correctly, you will see a Julia banner and an
```

Comment:
This text seems pretty superfluous.

Reply:
Yeah, I guess it can go. Just pointing to the manual is sufficient.